specification/dimensions/tags.md

```diff
@@ -0,0 +1,64 @@
+# Tags
+
+Tags are the set of finalized User-Defined and/or Provider-Defined Tags assigned to Tag Sources.  Tags commonly add business context to billing data to identify and accurately allocate charges.
+
+If a Tag is subject to a set of prioritized rules for determining its value, it is finalized after a single value is chosen from a set of corresponding values.  Otherwise, a Tag and a finalized Tag are equal.
+
+Tags adhere to the following requirements:
+
+* Tags MUST only contain finalized tags.
+* Tags MUST standardize to Key-Value Format.
+* A Tag without a specified value MUST have its value set to null.
+* Providers MUST NOT alter User-Defined Tag keys.
+
+Provider-defined Tags additionally adhere to the following requirements:
+
+* Provider-defined Tags MUST be prefixed with a provider-defined prefix.
+* Providers SHOULD publish all provider-defined prefixes within their respective documentation.
+
+## Provider-Defined vs. User-Defined Tags
+
+The following is an example of one User-Defined Tag and one Provider-Defined Tag, respectively, with tag key, foo.  The 1st property is not prefixed, and the Provider has a pre-selected prefix, `marketplace/`, applied to denote that it is a different type of tag.
```

Comment:
I'd suggest replacing "tag key, foo" with "tag key `foo`" and replace "The 1st property" with "The first property"